tests/bsim/bluetooth/mesh/src/test_transport.c

```diff
@@ -478,6 +504,60 @@ static void test_rx_group(void)
 	PASS();
 }
 
+/** Test that a node delivers a message to a model subscribed to a fixed group address even if the
+ *  corresponding feature is disabled.
+ */
+static void test_rx_fixed(void)
+{
+	uint8_t status;
+	int err;
+
+	bt_mesh_test_setup();
+
+	/* Step 1.
+	 *
+	 * The model is on primary element, so it should receive the message if the Relay feature
+	 * is enabled.
+	 */
+	err = bt_mesh_relay_set(BT_MESH_RELAY_ENABLED,
+				BT_MESH_TRANSMIT(CONFIG_BT_MESH_RELAY_RETRANSMIT_COUNT,
+						 CONFIG_BT_MESH_RELAY_RETRANSMIT_INTERVAL));
+	ASSERT_EQUAL(err, -EALREADY);
```

Comment:
```suggestion
	ASSERT_TRUE(err == -EALREADY || err == 0);
```
The test will pass despite anybody initially disabling the relay feature or doesn't.

Reply:
I prefer here explicit check, because `ASSERT_TRUE` won't print error code. I'd also prefer to check only against `EALREADY` only.